build_config/README.md

```diff
@@ -123,6 +124,57 @@ builders:
     auto_apply: dependents
 ```
 
+## Defining `PostProcessBuilder`s
+
+`PostProcessBuilder`s are configured similarly to normal `Builder`s, but they
+have some different/missing options.
+
+These builders can not be auto-applied on their own, and must always build to
+cache because their outputs are not declared ahead of time. To apply them a
+user will need to explicitly enable them on a target, or a `Builder` definition
+can add them to `apply_builders`.
+
+Exposed `PostProcessBuilder`s are configured in the `post_process_builders`
+section of the  `build.yaml`. This is a map of builder names to configuration.
+Each post process builder config may contain the following keys:
+
+- **target**: The name of the target which defines contains the `Builder` class
+  definition.
+- **import**: Required. The import uri that should be used to import the library
+  containing the `Builder` class. This should always be a `package:` uri.
+- **builder_factories**: A `List<String>` which contains the names of the
```

Comment:
discussed offline that this should be `builder_factory`. Having multiple factories isn't necessary cause it doesn't buy any functionality like it does with normal builders where you can see previous outputs. Better to not make it look like it might.

Reply:
done